sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Trigger/WebPubSubTriggerDispatcher.cs

```diff
@@ -201,6 +232,13 @@ private static bool TryParseCloudEvents(HttpRequestMessage request, out WebPubSu
                 {
                     states = connectionStates.SingleOrDefault().DecodeConnectionStates();
                 }
+                if (request.Headers.TryGetValues(Constants.Headers.CloudEvents.Subprotocol, out var subprotocols) && subprotocols.Contains(Constants.MqttWebSocketSubprotocolValue)
```

Comment:
wrap it into some util method like `SupportedMqttRequest` and reuse it?

Reply:
The `Header` types are different in WPSTriggerDispatcher and WPSRequestExtensions. One is `IHeaderDictionary` and the other one is `HttpRequestHeaders`. Cannot be reused.